client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/task-helper.ts

```diff
@@ -270,6 +271,23 @@ export function getEnhancedTasks(
 						},
 					};
 					break;
+				case 'sensei_setup':
+					taskData = {
+						title: translate( 'Set up Course Site' ),
+						completed: true,
+					};
+					break;
+				case 'sensei_publish_first_course':
+					taskData = {
+						title: translate( 'Publish your first Course' ),
+						completed:
+							site?.options?.launchpad_checklist_tasks_statuses?.publish_first_course || false,
```

Comment:
The `publish_first_course` launchpad_checklist_tasks_statuses property can also be updated in WPCOM (launchpad.php), so the correct value will be reflected here.

Reply:
We are going to drop the launchpad screen from our flow so we will handle this as part of another card.